README.md

```diff
@@ -1,4 +1,4 @@
-[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test.yml)
+[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test-mac.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test-mac.yml)
```

Comment:
Please add the fedora and debian links here. The debian one is the most important.

Reply:
I would make sure to add that. Thanks for letting me know.